scripts/build-prep-1node.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/sh
+
```

Comment:
Is the license header required for shell scripts? I see some scripts do put the license, some are not, what is the rule in cort-motr?

Reply:
Yes, let me add it...

Reply:
Added.